README.md

```diff
@@ -57,4 +57,4 @@ You can then use the functionality provided by `xrpl.js` through the `xrpl.dart`
 Contributions to xrpl.dart are welcome! To contribute, please fork the repository and submit a pull request. Before submitting a pull request, please make sure that your code follows the [Dart Style Guide](https://dart.dev/guides/language/effective-dart/style).
 
 ## license
-xrpl.dart is licensed under the MIT License. See the LICENSE file for more information.
\ No newline at end of file
+xrpl.dart is licensed under the MIT License. See the LICENSE file for more information.
```

Comment:
It's not clear what has changed here

Reply:
Not sure, but this line is also incorrect!  I'll add another commit to fix it

Reply:
The first commit (introducing this part of the diff) should be fixed now, and I've added another to fix the incorrect MIT license reference in the `README.md`